client_side/.infrastructure/infrastructure.sh

```diff
@@ -200,18 +200,25 @@ check_and_update_training_status() {
 # - The directory that they should be performing tasks on.
 infra_training() {
   echo ${HLINE}
-  echo "For this task, please follow the instructions at"
-  echo "${EXPERIMENT_HOME_URL}/experiment.html"
-  echo "to learn how to do the experiment."
+  echo "This terminal is the interface for the experiment."
+  echo "At any point, run \"helpme\" to see a list of commands available to you."
+  echo ""
+  echo "For each task, we ask you to a corresponding bash one-liner satisfying the prompt, just like you would ordinarily."
```

Comment:
"we ask you to (is something missing here?) a corresponding ..."

Reply:
Good catch! Thank you 🙏